src/borg/testsuite/helpers.py

```diff
@@ -1022,3 +1025,28 @@ def test_dash_open():
     assert dash_open('-', 'w') is sys.stdout
     assert dash_open('-', 'rb') is sys.stdin.buffer
     assert dash_open('-', 'wb') is sys.stdout.buffer
+
+
+def test_iter_separated():
+    # newline and utf-8
+    sep, items = '\n', ['foo', 'bar/baz', '나윤a선나윤선나윤선나윤선나윤선']
```

Comment:
my font does not really display characters here, just squares with numbers.

maybe use more usual unicode chars? äöü?

Reply:
You don't have a CJK font installed? :) Ok i changed some, i just left an α character since else we only have accentuated characters, which might not be very representative.